develop/reference_dml.rst

```diff
@@ -149,6 +149,8 @@ The situation changes when dealing with late arriving data, or running the rollu
     ON CONFLICT (day, url, site_id) DO UPDATE SET
       view_count = daily_page_views.view_count + EXCLUDED.view_count;
 
+(In reality this particular construction needs a slight modification on Citus, see the :ref:`upsert_into_select` workaround.)
```

Comment:
Only Insert/Selects which don't get pushed down entirely need the construct. Do you know if this query needs to be pulled up? I think site_id is the partition column, so it seems this query will just work. Could you try it out?

Reply:
You're right, it worked. I should have tested it earlier.